icu4c/source/common/uniset_props.cpp

```diff
@@ -105,13 +105,34 @@ static UBool U_CALLCONV uset_cleanup(void) {
     return TRUE;
 }
 
+#ifdef __SUNPRO_CC
+void U_CALLCONV createUni32Set(UErrorCode &errorCode) {
```

Comment:
If these are outside of the anonymous namespace, should they be marked as `static`?